src/test/java/service/FruitShopServiceTest.java

```diff
@@ -0,0 +1,68 @@
+package service;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import model.FruitTransaction;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import service.impl.FruitShopServiceImpl;
+import strategy.OperationHandler;
+import strategy.impl.BalanceOperationImpl;
+import strategy.impl.OperationStrategyServiceImpl;
+import strategy.impl.PurchaseOperationImpl;
+import strategy.impl.ReturnOperationImpl;
+import strategy.impl.SupplyOperationImpl;
+
+public class FruitShopServiceTest {
+    private static final String BANANA = "banana";
+    private static final String APPLE = "apple";
+    private static FruitShopService defaultFruitShopService;
+    private static List<FruitTransaction> defaultParsed;
+    private static Map<String, Integer> defaultExpected;
+
+    @BeforeClass
+    public static void setUp() {

```

Comment:
Don't forget, that you have a direct access to Storage, and you can check it. Note: Don't forget to clean it after every test.